assets/js/player.js

```diff
@@ -59,6 +49,38 @@ videojs.Vhs.xhr.beforeRequest = function(options) {
 
 var player = videojs('player', options);
 
+/**
+ * Function for add time argument to url
+ * @param {String} url 
+ * @returns urlWithTimeArg
+ */
+function addCurrentTimeToURL(url) {
+    var urlUsed = new URL(url);
+    urlUsed.searchParams.delete('start');
+    var currentTime = Math.ceil(player.currentTime());
+    if (currentTime > 0)
+        urlUsed.searchParams.set('t', currentTime);
+    else
+        if (urlUsed.searchParams.has('t'))
```

Comment:
Is this check necessary? You don't seel to check if the `start` parameter exists just above.

Reply:
Yes, here it is necessary because if for example we have a `t` argument defined, it will always be present while the player is at the beginning of the video. In this case it must be removed (in addition to the start argument which is systematically removed to avoid having 2 arguments specifying the starting point [Example : `start=1m2&t=30`]) otherwise we may find ourselves in a situation where we give t=30 when we want to share from the beginning.

Reply:
Could you just inline the `else if` then? (just a matter of style, and trying to reduce the amoun of useless indentations, as the JS is not minified).

Reply:
Of course! (By the way, since it is mentioned, is there any plan to minify the JS ? [Because I confess I haven't seen any issue in this sense and that it has an interest in reducing the weight of the page])

Reply:
We won't minify the JS in order to keep it legible by an end user. The only JS that can be minified are the VideoJS dependencies (thought it's not the case by default).

While invidious is proxying video data (= a lot of bytes), I'm in favor of trying to minimize the amounf of data sent to the user at other places, in order to reduce the stress on large instances (if we can save a kilobyte on HTML and JS, that can be a lot when an instance gets a million requests a day).

Reply:
Okay, I didn't know this was voluntary. In this case I understand even more the desire to reduce the weight of assets (Scripts, etc)